content/src/controller/Controller.ts

```diff
@@ -396,7 +392,7 @@ export class Controller {
       pointerChanges: deltas,
       filters,
       pagination
-    } = await this.components.deployer.getPointerChanges(undefined, {
+    } = await this.components.deployer.getPointerChanges({
```

Comment:
the function `getPointerChanges` can be safely removed from the interface of the deployer to start decoupling the code

```ts
import { getPointerChanges } from './deployments/deployments'
```

```suggestion
    } = await getPointerChanges(this.components, {
```